classes/class-wc-connect-tracks.php

```diff
@@ -100,11 +86,11 @@ public function record_user_event( $event_type, $data = array() ) {
 			$data['_via_ua']         = isset( $_SERVER['HTTP_USER_AGENT'] ) ? wc_clean( wp_unslash( $_SERVER['HTTP_USER_AGENT'] ) ) : '';
 			$data['_via_ip']         = isset( $_SERVER['REMOTE_ADDR'] ) ? wc_clean( wp_unslash( $_SERVER['REMOTE_ADDR'] ) ) : '';
 			$data['_lg']             = isset( $_SERVER['HTTP_ACCEPT_LANGUAGE'] ) ? wc_clean( wp_unslash( $_SERVER['HTTP_ACCEPT_LANGUAGE'] ) ) : '';
-			$data['blog_url']        = $site_url;
+			$data['blog_url']        = get_option( 'siteurl' );
 			$data['blog_id']         = $jetpack_blog_id;
-			$data['wcs_version']     = $wcs_version;
-			$data['jetpack_version'] = $jp_version;
-			$data['is_atomic']       = $is_atomic;
+			$data['wcs_version']     = WC_Connect_Loader::get_wcs_version();
+			$data['jetpack_version'] = 'embed-' . WC_Connect_Jetpack::get_jetpack_connection_package_version();
```

Comment:
What does the `embed-` prefix do?

Reply:
This is just something to distinguish values from back when this used to be the version of Jetpack-the-plugin in use and now that it is the version of the embedded Jetpack Connection package.